app/controllers/facility_notifications_controller.rb

```diff
@@ -46,6 +46,10 @@ def send_notifications
 
     if sender.perform
       flash[:notice] = send_notification_success_message(sender)
+      order_details = OrderDetail.where(id: params[:order_detail_ids])
```

Comment:
You probably haven't run into it before, but something to be aware of is Oracle has a 1000 item limit in `WHERE IN` statements. So this query would fail if we're sending notifications to more than 1000 order details. We have a special scope we've created to deal with this `where_ids_in` that we would want to use.

In order to completely avoid the issue, can we get the list of order details out of the `sender` rather than doing an additional query? It looks like `NotificationSender#order_details` is private, but I wouldn't have a problem with making it public.

In that sender, there's a `@orders_notified` variable that I don't think is used. If it's not used, can we get rid of it.

Reply:
I'll take a look and change it